e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -293,69 +295,112 @@ public void testDelete() throws IOException, InterruptedException {
+        HttpRequest exportRequest = HttpRequest.newBuilder()
+                .uri(URI.create(AB2D_API_URL + PATIENT_EXPORT_PATH))
+                .timeout(Duration.ofSeconds(30))
+                .header("Content-Type", "application/json")
+                .GET()
+                .build();
+
+        HttpResponse<String> response = apiClient.getHttpClient().send(exportRequest, HttpResponse.BodyHandlers.ofString());
+
+        Assert.assertEquals(401, response.statusCode());
+    }
+
+    @Test
+    public void testUserCannotMakeRequestWithSelfSignedToken() throws IOException, InterruptedException, JSONException {
+        String clientSecret = "wefikjweglkhjwelgkjweglkwegwegewg";
+        SecretKey sharedSecret = Keys.hmacShaKeyFor(clientSecret.getBytes(StandardCharsets.UTF_8));
+        Instant now = Instant.now();
+
+        String jwtStr = Jwts.builder()
+                .setAudience(System.getenv("AB2D_OKTA_JWT_AUDIENCE"))
+                .setIssuedAt(Date.from(now))
+                .setExpiration(Date.from(now.plus(2L, ChronoUnit.HOURS)))
+                .setIssuer(System.getenv("AB2D_OKTA_JWT_ISSUER"))
+                .setId(UUID.randomUUID().toString())
+                .signWith(sharedSecret)
+                .compact();
+
+        HttpRequest exportRequest = HttpRequest.newBuilder()
+                .uri(URI.create(AB2D_API_URL + PATIENT_EXPORT_PATH))
+                .timeout(Duration.ofSeconds(30))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + jwtStr)
+                .GET()
+                .build();
+
+        HttpResponse<String> response = apiClient.getHttpClient().send(exportRequest, HttpResponse.BodyHandlers.ofString());
+
+        Assert.assertEquals(500, response.statusCode());
+
+        final JSONObject json = new JSONObject(response.body());
+        JSONArray issueJsonArr = json.getJSONArray("issue");
+        JSONObject issueJson = issueJsonArr.getJSONObject(0);
+        JSONObject detailsJson = issueJson.getJSONObject("details");
+        String text = detailsJson.getString("text");
+
+        Assert.assertEquals(text, "IllegalArgumentException: A signing key must be specified if the specified JWT is digitally signed.");
+    }
+
+    @Test
+    public void testBadQueryParameterResource() throws IOException, InterruptedException {
+        var params = new HashMap<>(){{
+            put("_type", "BadParam");
+        }};
+        HttpResponse<String> exportResponse = apiClient.exportRequest(params);
+
+        Assert.assertEquals(400, exportResponse.statusCode());
+    }
+
+    @Test
+    public void testBadQueryParameterOutputFormat() throws IOException, InterruptedException {
+        var params = new HashMap<>(){{
+            put("_outputFormat", "BadParam");
+        }};
+        HttpResponse<String> exportResponse = apiClient.exportRequest(params);
+
+        Assert.assertEquals(400, exportResponse.statusCode());
+    }
```

Comment:
Given the earlier comments, we need two more test cases:
- User cannot check status of a job that does not belong to them
- User cannot cancel other's job